apps/console/src/features/server-configurations/components/governance-connectors/dynamic-governance-connector.tsx

```diff
@@ -45,9 +48,9 @@ interface DynamicGovernanceConnectorProps extends TestableComponentInterface, Id
 /**
  * Dynamic governance connector component.
  *
- * @param {DynamicGovernanceConnectorProps} props - Props injected to the dynamic connector component.
+ * @param props - Props injected to the dynamic connector component.
  *
- * @return {React.ReactElement}
+ * @returns \{React.ReactElement\}
```

Comment:
Shall we remove `\` and update the comment to be consistent with other places?

Reply:
Fixed by https://github.com/wso2/identity-apps/pull/3982/commits/8fd831d8584b09497fbab6218a7b567d712d87df